.test-infra/jenkins/job_LoadTests_CoGBK_Dataflow_V2_Java11.groovy

```diff
@@ -47,6 +47,7 @@ def loadTestConfigurations = { mode, isStreaming ->
             "numRecords": 20000000,
             "keySizeBytes": 10,
             "valueSizeBytes": 90,
+            "forceNumInitialBundles": 1,
```

Comment:
Could you add a comment with TODO(BEAM-14429) here and in other places?

Reply:
Done.